man/unison.1.in

```diff
@@ -310,9 +310,23 @@ or
 .Sy SIGTERM
 works the same way as described above and will additionally stop the continuous
 process. To stop only the continuous process and let the last synchronization
-complete normally, send signal
+complete normally, press
+.Sy Ctrl-D
+or send signal
 .Sy SIGUSR2
 instead.
+Closing the input, receiving an EOF, or receiving a
+.Sy ^D
+(0x04) from the input terminal or a redirected standard input all have the same
+effect. If the standard input is redirected from a regular file, is not open or
+is not open for reading already at Unison startup then it is ignored and only
+signals can be used to send the stop request.
+.Bd -ragged -offset indent
+Tips: For continuous synchronization the input should not be redirected from
+any source providing a lot of input (such as a device or a pipe that produces
+large quantities of data). In such cases, Unison would keep reading all the
+input looking for the stop condition.
+.Ed
```

Comment:
True, but it seems that whatever is provided for stdin should produce essentially nothing until it either sends EOF or 0x4, and anything else is odd.

Reply:
It is somewhat unnecessary, yes. I'll try to rephrase or get rid of it completely. I thought about warning users not to shoot themselves in the foot but the same is true for any stdin redirection, so nothing special here.

Reply:
ok, just dropping it seems fine.